com.unity.multiplayer.mlapi/Tests/Runtime/RpcQueueTests.cs

```diff
@@ -14,99 +14,105 @@ namespace MLAPI.RuntimeTests
     /// That all RPCs invoke at the appropriate NetworkUpdateStage (Client and Server)
     /// A lower level RpcQueueContainer test that validates RpcQueueFrameItems after they have been put into the queue
     /// </summary>
-    public class RpcQueueTests : IDisposable
+    public class RpcQueueTests
     {
+        [SetUp]
+        public void Setup()
+        {
+            // Create, instantiate, and host
+            Assert.IsTrue(NetworkManagerHelper.StartNetworkManager(out _));
+        }
+
         /// <summary>
-        /// Tests to make sure providing differen
+        /// Tests to make sure providing different
         /// ** This does not include any of the MLAPI to Transport code **
         /// </summary>
         /// <returns>IEnumerator</returns>
-        [UnityTest]
+        [UnityTest,Order(1)]
         public IEnumerator UpdateStagesInvocation()
         {
-            Assert.IsTrue(NetworkManagerHelper.StartNetworkManager(out _));
-
             Guid updateStagesTestId = NetworkManagerHelper.AddGameNetworkObject("UpdateStagesTest");
             var rpcPipelineTestComponent = NetworkManagerHelper.AddComponentToObject<NetworkUpdateStagesComponent>(updateStagesTestId);
+      
             NetworkManagerHelper.SpawnNetworkObject(updateStagesTestId);
+            
             var testsAreComplete = rpcPipelineTestComponent.IsTestComplete();
             var exceededMaximumStageIterations = rpcPipelineTestComponent.ExceededMaxIterations();
 
-            //Start testing
+            // Start testing
             rpcPipelineTestComponent.EnableTesting = true;
 
             Debug.Log("Running TestNetworkUpdateStages: ");
 
-            //Wait for the rpc pipeline test to complete or if we exceeded the maximum iterations bail
+            // Wait for the rpc pipeline test to complete or if we exceeded the maximum iterations bail
             while (!testsAreComplete && !exceededMaximumStageIterations)
             {
                 //Wait for 20ms
-                yield return new WaitForSeconds(0.02f);
+                yield return new WaitForSeconds(0.003f);
```

Comment:
Please update the comment above that says 20ms :-)

Reply:
Hahah yeah, nice catch!